buildkite/scripts/cache-through.sh

```diff
@@ -10,7 +10,9 @@ fi
 
 # download gsutil if it doesn't exist
 # TODO: Bake this into the agents
```

Comment:
this TODO should be fixed with: https://github.com/CodaProtocol/coda-automation/pull/386/files#diff-c8dd0de851b40328081ea6779d3842c9R64